cdc/owner/feed_state_manager.go

```diff
@@ -238,7 +269,8 @@ func (m *feedStateManager) handleAdminJob() (jobsPending bool) {
 			zap.Uint64("checkpointTs", checkpointTs))
 	case model.AdminResume:
 		switch m.state.Info.State {
-		case model.StateFailed, model.StateError, model.StateStopped, model.StateFinished:
```

Comment:
if the state is  warning or pending ,whether  it can be resumed ?

Reply:
yes,it can be resumed.